build/bin/kylin.sh

```diff
@@ -40,16 +40,10 @@ function retrieveDependency() {
         source ${dir}/cached-hive-dependency.sh
         source ${dir}/cached-hbase-dependency.sh
         source ${dir}/cached-hadoop-conf-dir.sh
-        source ${dir}/cached-kafka-dependency.sh
```

Comment:
Why should remove these lines?

Reply:
Because these three checks are put in check-env.sh, there is no need to check them again here.
But now it is found that some tool commands, such as `bin/kylin.sh org.apache.kylin.source.kafka.util.KafkaSampleProducer`, require Kafka's dependency. However, the check-env.sh is not executed when using this command, which makes it unable to load Kafka's dependency. 
This is my negligence. It should not be removed here.